app/records-interfaces/group-records-interface.coffee

```diff
@@ -1,6 +1,9 @@
-null # ridiculous hack due to coffee class extends
+null
```

Comment:
are we going to remember why this ridiculous hack is here? or if we on-board a new dev, how will they know?